Sources/MediaEventTracker.swift

```diff
@@ -109,9 +109,9 @@ class MediaEventTracker: MediaEventTracking {
 
     private static let LOG_TAG = MediaConstants.LOG_TAG
     private static let CLASS_NAME = "MediaEventTracker"
-    private static let IDLE_TIMEOUT_MS: Int64 = 1800 * 1000 // 30 min
-    private static let MEDIA_SESSION_TIMEOUT_MS: Int64 = 86400 * 1000 // 24 hours
-    private static let CONTENT_START_DURATION_MS: Int64 = 1 * 1000 // 1 sec
+    private static let IDLE_TIMEOUT_S: Int64 = 1800 // 30 min
+    private static let MEDIA_SESSION_TIMEOUT_S: Int64 = 86400 // 24 hours
+    private static let CONTENT_START_DURATION_S: Int64 = 1 // 1 sec
```

Comment:
I'm wondering if you changed these to be of type `TimeInterval` would it eliminate the need to type-cast to Int when comparing to the various timestamp variables?